src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -843,8 +843,30 @@ winrt::Windows::Foundation::IAsyncAction AppHost::_SaveWindowLayouts()
 
     if (_logic.ShouldUsePersistedLayout())
     {
-        const auto layoutJsons = _windowManager.GetAllWindowLayouts();
-        _logic.SaveWindowLayoutJsons(layoutJsons);
+        try
+        {
+            TraceLoggingWrite(g_hWindowsTerminalProvider,

```

Comment:
these seem somewhat unrelated, but nice

Reply:
lol meant to pull these out to a separate PR for 1.12, whoops